src/translations/index.js

```diff
@@ -0,0 +1,7 @@
+import english from "./english";
```

Comment:
can you use this for the `/location` page as well?

Reply:
done